src/renderer/components/dialogs/index.js

```diff
@@ -19,14 +19,16 @@ module.exports = {
   QrCode
 }
 
-function confirmation (message, cb) {
+function confirmation (message, opts, cb) {
+  if (!cb) cb = opts
+  if (!opts) opts = {}
   const tx = window.translate
-  const opts = {
+  var defaultOpts = {
     type: 'question',
     message: message,
     buttons: [tx('dialogs.confirmation.no'), tx('dialogs.confirmation.yes')]
   }
-  remote.dialog.showMessageBox(opts, response => {
+  remote.dialog.showMessageBox(Object.assign(defaultOpts, opts, {}), response => {
```

Comment:
Why the last `{}`?

Reply:
Did you mean to do `opts || {}`?

Reply:
creates a copy I think although perhaps not necessary? does Object.assign mutate?

Reply:
Yes it mutates first argument.